src/utils/schema.js

```diff
@@ -14,10 +14,14 @@ exports.getModelName = (model) => model.modelName;
 // TODO: Remove nameOld attribute once the lianas versions older than 2.0.0 are minority
 exports.getModelNameOld = (model) => model.collection.name.replace(' ', '');
 
+const Flattener = require('../services/flattener');
+
+const { FLATTEN_SEPARATOR } = Flattener;
```

Comment:
Little suggestion, and this should on top as-well

```suggestion
const { FLATTEN_SEPARATOR } = require('../services/flattener');
```